bindings/python/examples/client/10_mint_nft.py

```diff
@@ -28,7 +35,7 @@
     nft_id='0x0000000000000000000000000000000000000000000000000000000000000000',
     amount=1000000,
     immutable_features=[
-        MetadataFeature(utf8_to_hex('Hello, World!'))
+        MetadataFeature(metadata.as_hex())
```

Comment:
Would it be handy to also have an `as_feature()` method ?

Reply:
could be yeah

Reply:
Also if ever we follow rust conventions in python, it would rather be `to` than `as` but whatever

Reply:
@DaughterOfMars Making sure you've seen the last comment, just resolve yourself if you want to ignore it